src/controllers/tripsController.js

```diff
@@ -107,7 +107,7 @@ export default class requestController {
           passportName,
           role
         }],
-        status: 'Open'
+        status: 'open',
```

Comment:
which case did we decide to go with?

Reply:
lower case is much better so that's what we are going with